### DISCUSSION TITLE
Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by laoaaxf** [microrealestate/microrealestate#199] [Q&A]
Everything I followed the default installation method of
cd microrealestate
./mre start.

I originally left the URL as HTTP://localhost:8080/landlord
But this made it so that I could not sign up for some reason.
I decided to redownload it and use HTTP://192.168.1.54:8080/landlord and everything was working.

In NPM I used mre.domain.net
HTTP 192.168.1.54 port 8080.
It allows me to get to the webpage at mre.domain.net/landlord.

But I get these logs 

info: OPTIONS /landlord/signin 200 0ms

info: OPTIONS /landlord/signin 200 0ms

info: OPTIONS /landlord/signin 200 0ms

info: OPTIONS /landlord/signin 200 1ms

info: OPTIONS /landlord/signin 200 1ms

debug: give a new refresh token for undefined

info: POST /landlord/refreshtoken 403 0ms

from the auth container in docker.

**Comment by camelaissani** [comment]
I just updated the install procedure: 

https://github.com/microrealestate/microrealestate?tab=readme-ov-file#self-host-the-application

Tell me if it works.

**Reply by laoaaxf** [reply]
I got the proxy to work without the new update by going into portainer and reloading the container with 
DOMAIN_URL=mre.domain.net
CORS_ENABLED=true

Except now it stops working after a day or so and no longer able to login. Looking at the logs it seems to be a refresh token. Authenticator container keeps logging "please provide new refresh token"

For the new install I tried using NPM. It did the same thing and doing the steps from above did also solve the problem with NPM.

I did the new install in another docker lxc in proxmox and the UI looks much cleaner. Is there plans to allow smb or nfs storage also? Uploading from a NAS and storing data there would be awesome.  

Also I am having trouble using the dumpdb command. It ask for a dev and test file in cd microrealestate. I don't seem to be able to find a way to keep my data that's in the app.
Also in the new install I cant find a way to backup.
I would like to update the app and keep my data somehow.
